TrafficCrash/Pages/Index.cshtml.cs

```diff
@@ -27,15 +37,19 @@ public JsonResult OnGet()
             List<Vehicle_People_Crash_details> vehicle_n_people = new List<Vehicle_People_Crash_details>();
             using (WebClient webClient = new WebClient())
             {
-                string vehicleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/68nd-jvt3.json");
+     //           string vehicleJson = webClient.DownloadString("https://data.cityofchicago.org/resource/68nd-jvt3.json");
```

Comment:
Remove this entirely.  It's considered bad practice to commit commented-out code to version control, because you can always go back in history to see what used to be there.